src/main.rs

```diff
@@ -77,15 +77,19 @@ struct Args {
     flag_version: bool,
 }
 
+fn exit_on_err(message: String) -> ! {
```

Comment:
I think, you normally would just want to have `message: &str` as the parameter, but that requires the function to be called like this

```rust
exit_on_err(&format!("Could not {} file {}.", action, filepath));
```
Which looks a bit more ugly.